crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -878,7 +878,7 @@
   "components": {
     "schemas": {
       "AuthorityQuorumSignInfo": {
-        "description": "Represents at least a quorum (could be more) of authority signatures.",
+        "description": "Represents at least a quorum (could be more) of authority signatures. STRONG_THRESHOLD indicates whether to use the quorum threshold for quorum check. When StrongThreshold is true, the quorum is valid when the total stake is at least the quorum threshold of the committee; when STRONG_THRESHOLD is false, the quorum is valid when the total stake is at least the validity threshold of the committee.",
```

Comment:
StrongThreshold -> STRONG_THRESHOLD

Reply:
Can we clarify the difference between quorum threshold and validity threshold in the comment?. Adding a parenthesis would work. i.e.

> STRONG_THRESHOLD indicates whether to use the quorum threshold for quorum check. When STRONG_THRESHOLD is true, the quorum is valid when the total stake is at least the quorum threshold **(2f+1)** of the committee; when STRONG_THRESHOLD is false, the quorum is valid when the total stake is at least the validity threshold **(f+1)** of the committee.